quickstart-mongodbatlas/ci/mongodb-master-2AZ.json

```diff
@@ -0,0 +1,34 @@
+[
```

Comment:
This left over from the current quickstart or something required for taskcat?

Reply:
yes it is leftover, I will just chuck it for now.
we can add back later when AWS pulls